core/src/main/java/bisq/core/trade/protocol/ProcessModel.java

```diff
@@ -305,59 +266,118 @@ public void setTakeOfferFeeTx(Transaction takeOfferFeeTx) {
     public PaymentAccountPayload getPaymentAccountPayload(Trade trade) {
```

Comment:
This is not new in this PR, but would be better named `getMyPaymentAccountPayload`

Reply:
ProcessModel always contains my data, if its peers data its in the tradingPeer model.